Update code

Tokaage opened the pull request.

Tokaage committed:
Update code

KiraKrabu commented on code:
```diff
@@ -22,3 +25,84 @@ int main() {
+	}
+	return false;
+}
+std::vector<int> Deviders(int number) {
```
если ты указал include <vector> std можно же убрать?

Tokaage committed:
Переделанный код

учёл все комментарии, решил переписать код. В Настином коде убрал строку "else count += 0;", так как она не нужна, без неё всё работает прекрасно

Tokaage committed:
Переделанный код 1

в коде до этого, случайно удалил не то, вернул на место
